_config/oembed.yml

```diff
@@ -4,6 +4,13 @@ Name: coreoembed
 SilverStripe\Core\Injector\Injector:
   SilverStripe\View\Embed\Embeddable:
     class: SilverStripe\View\Embed\EmbedContainer
+  # Use guzzle instead of the default Embed\Embed curl implementation so that
+  # a proxy url can be set via yml if required
+  Psr\Http\Client\ClientInterface.oembed:
+    class: GuzzleHttp\Client
+  Embed\Http\Crawler:
+    constructor:
+      - '%$Psr\Http\Client\ClientInterface.oembed'
```

Comment:
It's a good idea using Guzzle all the time.

Reply:
I'm thinking maybe in some later PR we'll want to define a global HTTP client that people can use for every request. But that's another PR for another day.